packages/terra-framework-docs/src/terra-dev-site/test/menu/menu-item/MenuItemsWithIcons.test.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Menu from 'terra-menu';
+import IconAlert from 'terra-icon/lib/icon/IconAlert';
+import styles from './MenuItemCommon.module.scss';
+
+const cx = classNames.bind(styles);
+
+const childContextTypes = {
+  isSelectableMenu: PropTypes.bool,
+};
+
+class MenuItemWithIcon extends React.Component {
+  getChildContext() {
+    return { isSelectableMenu: true };
+  }
+
+  render() {
+    return (
+      <div>
+        <p>default icon, isToggled, isToggleable, and isInstructionsForUse combinations</p>
+        <br />
+        <ul className={cx('unordered-list')} role="menu">
+          <Menu.Item
+            text="Icon"
+            key="Icon"
+            onClick={this.handleOnClick}
```

Comment:
onClick is not used in this example !!

Reply:
removed:
https://github.com/cerner/terra-framework/pull/1592/commits/da3cde9689c50e0bec166f69b4dad6f086b09350